src/cpp/redisserver.cpp

```diff
@@ -26,74 +26,68 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <ctype.h>
 #include "redisserver.h"
 
 using namespace SmartRedis;
 
+// We should never seed srand more than once. There are more elegant ways
+// to prevent it, but this will suffice
+static bool ___srand_seeded = false;
```

Comment:
I think a better solution might be to make something like a ``std::default_random_engine`` as a member variable and then init that seed upon construction of redisserver.